services/devicesService.go

```diff
@@ -0,0 +1,27 @@
+package services
+
+import (
+	"tsm/models"
+	"tsm/repositories"
+)
+
+var (
+	DeviceService deviceServiceInterface = &deviceService{}
+)
+
+type deviceServiceInterface interface {
+	CreateDevice(*models.Device) (*models.DeviceWithId, models.MessageErr)
+}
+
+type deviceService struct{}
```

Comment:
I have already mentioned some things about public struct; public global state (`var DeviceService`) in repository - I would stick to the same rule here. In short, remove `deviceServiceInterface`, create `DeviceRepository` interface, remove global `var DeviceService deviceServiceInterface`.

I would change one more thing here - provide DeviceRepo via dependency injection (manual). Let's change `deviceService` struct into:
```golang
type deviceService struct{
  DeviceRepo DeviceRepo
}
```

This way you can still inject implementation (real, or mock) without even importing repositories package.